cluster/vagrant/provision-utils.sh

```diff
@@ -113,8 +119,10 @@ function install-salt() {
   popd
 
   if ! which salt-call >/dev/null 2>&1; then
-    # Install salt binaries
-    curl -sS -L --connect-timeout 20 --retry 6 --retry-delay 10 https://bootstrap.saltstack.com | sh -s
+    # Install salt from official repositories.
+    # Need to enable testing-repos to get version of salt with fix for dnf-core-plugins
+    dnf config-manager --set-enabled updates-testing
```

Comment:
Do you want to enable updates-testing forever here, or just do `dnf install --enablerepo=updates-testing -y salt-minion` below?